View/MainWindow.xaml.cs

```diff
@@ -20,9 +23,12 @@ namespace Soundify
     /// </summary>
     public partial class MainWindow : Window
     {
+        public DatabaseContext Context { get; set; }
+
         public MainWindow()
         {
             InitializeComponent();
+            Context = new DatabaseContext();
         }
     }
 }

```

Comment:
Should not be included in the models

Reply:
Nope that's init of the ef Database context

Reply:
Ok!